dockerfile/parser/parser.go

```diff
@@ -331,6 +404,26 @@ func Parse(rwc io.Reader) (*Result, error) {
 	}, nil
 }
 
+func heredocsFromLine(line string) ([]buildkitparser.Heredoc, error) {
```

Comment:
Did this previously use `reHeredoc` instead of calling out to new dependencies?  I can't see the regexp being used anywhere.

Reply:
Yes its not needed anymore.